apps/web/pages/booking/[uid].tsx

```diff
@@ -545,6 +545,14 @@ export default function Success(props: SuccessProps) {
                         </div>
                       </>
                     )}
+                    {bookingInfo?.responses?.title && typeof bookingInfo.responses.title === "string" && (
```

Comment:
Why do we need to do it separately ?  AFAIU we can get it to work by fixing this line to show this system field as well https://github.com/calcom/cal.com/blob/c6aab92ec57d55542aab7e58383b6659061d75aa/apps/web/pages/booking/%5Buid%5D.tsx#L567